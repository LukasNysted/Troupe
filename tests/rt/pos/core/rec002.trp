let val a = "Hello"
    val r = {a, b = 42, c = 10}
in (r.a, r.b + r.c)
end