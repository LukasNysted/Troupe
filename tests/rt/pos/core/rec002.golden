2020-10-16T12:42:50.160Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: ("Hello"@{}%{}, 52@{}%{})@{}%{}
